### PR TITLE
[refactor-prompt] update docs

### PR DESCRIPTION
**Pull request by ixje**
**What current issue(s) does this address, or what feature is it adding?**
 #623 changes a lot of cli commands and some output. This PR updates the `readthedocs` documentation.

**How did you solve this problem?**
update commands

**How did you make sure your solution works?**
`make docs` & browse a little

**Are there any special changes in the code that we should be aware of?**
no

**Please check the following, if applicable:**

- [ ] Did you add any tests?
- [ ] Did you run `make lint`?
- [ ] Did you run `make test`?
- [X] Are you making a PR to a feature branch or development rather than master?
- [ ] Did you add an entry to `CHANGELOG.rst`? (if not, please do)

### CODE REVIEW

File: docs/source/prompt.rst [hunk 1]
```diff
@@ -329,10 +321,17 @@ Next we create the multi signature address as follows.
...
+    Import a multi-signature address
+
+    Usage: wallet import multisig_addr {own pub key} {sign_cnt} {signing key n}
+
+    own pub key     - public key in your own wallet (use `wallet` to find the information)
+    sign_cnt        - minimum number of signatures required for using the address (min is: 1)
+    signing key n   - all remaining signing public keys
 
-    neo> import multisig_addr 037b8992e8384212f82e05c8836816c0f14dff9528397138731638b17d6357021e 1 037b8992e8384212f82e05c8836816c0f14dff9528397138731638b17d6357021e 02883118351f8f47107c83ab634dc7e4
+    neo> wallet import multisig_addr 037b8992e8384212f82e05c8836816c0f14dff9528397138731638b17d6357021e 1 02883118351f8f47107c83ab634dc7e4
```
[the remark this answers is not in the document]
Comment (ixje) [reply docs/source/prompt.rst]: #716 was before I changed it in #767 
I thought it made no sense that we had to enter our own key twice, so I changed it. Note how I here add our own public key (from our wallet) to the `signing_keys` before feeding it to `CreateMultiSigContract` call.
https://github.com/ixje/neo-python/blob/4906feb6c661a613f65c4bafd0a0df1388d3f23e/neo/Prompt/Commands/Wallet.py#L552-L564

File: docs/source/prompt.rst [hunk 2]
```diff
@@ -231,12 +223,12 @@ You may also specify a particular address to send assets from. This is especiall
 .. code-block:: sh
 
     # syntax send {asset_name} {address to} {amount} ( optional: --from-addr={from_addr})
-    neo> send gas AeU8kTJxynwkT3q9ao8aDFuaRJBkU3AfFG 11 --from-addr=AXjaFSP23Jkbe6Pk9pPGT6NBDs1HVdqaXK
+    neo> wallet sendfrom gas AeU8kTJxynwkT3q9ao8aDFuaRJBkU3AfFG 11 --from-addr=AXjaFSP23Jkbe6Pk9pPGT6NBDs1HVdqaXK
```
[the remark this answers is not in the document]
Comment (ixje) [reply docs/source/prompt.rst]: Good catch! There's a `sendfrom` in the `tokens` group which I probably had in mind when scanning the text and changing this.

File: docs/source/neo/SmartContract/smartcontracts.rst [hunk 3]
```diff
@@ -512,15 +507,15 @@ Now you have deployed your contract to the network. If all goes well, it will so
...
+  neo> show tx f8ad261d28bf4bc5544e47f9bc3fff85f85ee674f14162dac81dd56bf73cf0a3
   {
     "txid": "0xf8ad261d28bf4bc5544e47f9bc3fff85f85ee674f14162dac81dd56bf73cf0a3",
     "type": "InvocationTransaction",
     "version": 1,
     "attributes": [],
     [ MORE Output Omitted ]
 
-  neo> contract 0x86d58778c8d29e03182f38369f0d97782d303cc0
+  neo> show contract 0x86d58778c8d29e03182f38369f0d97782d303cc0
```
[the remark this answers is not in the document]
Comment (ixje) [reply docs/source/neo/SmartContract/smartcontracts.rst]: Yes, you're right. This is why I love having reviews 💪 . We catch things that others missed/did not have on their mind.

**Comment by ixje** [comment]
All good points. I scanned mostly for commands and their arguments that needed updating, but your review nicely caught a couple of improvements in the descriptions as well 👍 
I think I've addressed them all (except the `multisig_addr` one as that changed and works as described).

File: docs/source/prompt.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (jseagrave21) [reply docs/source/prompt.rst]: That makes a lot more sense!

**Comment by ixje** [comment]
we need the neo-boa update first, and that's out of my control

**Pull request merged**